spec/rails_translation_manager/cleaner_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+RSpec.describe RailsTranslationManager::Cleaner do
+
+  before(:each) do
+    FileUtils.cp('spec/locales/cleaner/with_whitespace.yml', 'spec/locales/cleaner/tmp')
```

Comment:
By convention, we usually make use of the `tmp` directory at the root of the repository. I'd be tempted to make a `tmp/.gitkeep` and [add the folder to `.gitignore`](https://github.com/alphagov/content-publisher/blob/61181bca488aea8855b56d3582bef55b426d5ff7/.gitignore#L12). Then you don't have to worry about calling `rm_rf` afterwards.

I'd also rename this `tmp` => `tmp.yml`. It doesn't matter a lot, but the missing file extension threw me for a moment, as it's not clear if you're working with a file or a directory.

Reply:
@ChrisBAshton I have created tmp/.gitkeep and removed after(:each) block.

Reply:
Thanks 👍 Ah I think I'd misunderstood the code before. I assumed you were copying the `with_whitespace.yml` file to a new file called `tmp`, but I think you were actually copying the file into a _directory_ called `tmp`. And then the `RailsTranslationManager::Cleaner` class takes a _directory_ (not a file) as an argument. I understand now!